flink-runtime-web/web-dashboard/package.json

```diff
@@ -7,27 +7,27 @@
   "devDependencies": {
     "browserify": "^9.0.3",
     "coffeeify": "^1.0.0",
-    "gulp": "^3.8.11",
+    "gulp": "^3.9.0",
     "gulp-browserify": "^0.5.1",
     "gulp-coffee": "^2.3.1",
     "gulp-coffeeify": "^0.1.8",
-    "gulp-concat": "^2.5.2",
-    "gulp-filter": "^2.0.2",
-    "gulp-jade": "^1.0.0",
-    "gulp-less": "^3.0.2",
-    "gulp-livereload": "^3.8.0",
-    "gulp-minify-css": "^1.0.0",
-    "gulp-ng-annotate": "^0.5.2",
-    "gulp-plumber": "^1.0.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-filter": "^3.0.1",
+    "gulp-jade": "^1.1.0",
+    "gulp-less": "^3.0.5",
+    "gulp-livereload": "^3.8.1",
+    "gulp-minify-css": "^1.2.1",
+    "gulp-ng-annotate": "^1.1.0",
+    "gulp-plumber": "^1.0.1",
     "gulp-rename": "^1.2.0",
-    "gulp-serve": "^0.3.1",
-    "gulp-sourcemaps": "^1.5.1",
-    "gulp-stylus": "^2.0.1",
-    "gulp-uglify": "^1.1.0",
-    "gulp-util": "^3.0.4",
+    "gulp-serve": "^1.2.0",
+    "gulp-sourcemaps": "^1.6.0",
+    "gulp-stylus": "^2.1.0",
+    "gulp-uglify": "^1.5.1",
+    "gulp-util": "^3.0.7",
     "jade": "^1.9.2",
     "jadeify": "^4.1.0",
-    "main-bower-files": "^2.6.2",
+    "main-bower-files": "^2.9.0",
```

Comment:
It is not an auto-generated file, it is basically the set build utils used for the dashboard. Think "Maven build plugins". It is an important part of the setup and should not be git-ignored.

Something is strange here, this should not be auto changed when running the build tools. The changes to this file should not be committed, they have nothing to do with the re-building of the dashboard files.

I am wondering if someone ran a util command that tries to bump all dependencies to the latest version.